torch/_inductor/codegen/cpp.py

```diff
@@ -1348,15 +1349,21 @@ def to_dtype(x, dtype, src_dtype=None):
             return f"cvt_fp32_to_lowp_fp<{DTYPE_TO_CPP[dtype]}>({x})"
         if opt_ctx_x.dtype in DTYPE_LOWP_FP and dtype in (torch.float, torch.float32):
             return f"cvt_lowp_fp_to_fp32<{DTYPE_TO_CPP[opt_ctx_x.dtype]}>({x})"
-        if opt_ctx_x.dtype == torch.uint8 and dtype in (torch.float, torch.float32):
+        if opt_ctx_x.dtype in (torch.uint8, torch.int8) and dtype in (
+            torch.float,
+            torch.float32,
```

Comment:
NIT: Not from this PR, but `torch.float` is an alias for `torch.float32` so only one should be needed.